config/initializers/job_configurations.rb

```diff
@@ -192,7 +192,7 @@
       monthly_key_metrics_report: {
         class: 'Reports::MonthlyKeyMetricsReport',
         cron: cron_24h,
-        args: -> { [Time.zone.today] },
+        args: -> { [Time.zone.yesterday] },
```

Comment:
this matches what we do for `DailyAuthsReport` and the daily reports used for metrics. Since the reports run at midnight UTC, this is the previous day. So on Oct 1, this will be Sep 30 now, which means the reports need to go until the *end* of that day rather than the day before it